### DISCUSSION TITLE
How to remove the Pinned Rows from the Global search?

### DISCUSSION THREAD

**Discussion by apoorvareddy** [KevinVandy/material-react-table#1052] [Q&A]
I am using the global search whenever i have some pinned items, if i search anything using global search irrespective of the search content whether it is there in the pinned rows or not but it is showing the pinned rows also in the search results. Can we exclude if it is not matching with the search string?

**Comment by KevinVandy** [comment]
Yeah, that's actually exactly what the `keepPinnedRows` table option is for. It's true by default, but you can set it to false.

**Reply by apoorvareddy** [reply]
Thanks a lot, can you help me to exclude the pinned rows from ordering also?
